habitat/core/vector_env.py

```diff
@@ -75,6 +76,30 @@ def _make_env_fn(
     return habitat_env
 
 
+@attr.s(auto_attribs=True, slots=True)
+class ReadWrapper:
```

Comment:
For such general wrappers doc strings will be useful.

Reply:
Happy to add doc strings, but these aren't really general purpose, they are just for the vector env implementation